alibi/api/tests/test_interfaces.py

```diff
@@ -8,6 +9,8 @@
 invalid_meta = []  # type: list
 invalid_data = {}  # type: dict
 
+data_dict = {'b': 2, 'c': 3, 'd': 4}
```

Comment:
This parameterizes the test so we need to add marks on the tests and pass it that way rather than relying on scope? This is mostly a matter of consistency with the rest of the library but also makes the tests easier to read/understand fast.